manifests/init.pp

```diff
@@ -661,16 +682,21 @@
     'RedHat' => ['overlay2.override_kernel_check=true'],
     default  => undef,
   },
-  Optional[String] $docker_extra_daemon_config            = undef,
-  String $docker_log_max_file                             = '1',
-  String $docker_log_max_size                             = '100m',
-  Boolean $disable_swap                                   = true,
-  Boolean $manage_kernel_modules                          = true,
-  Boolean $manage_sysctl_settings                         = true,
-  Boolean $create_repos                                   = true,
-  String $image_repository                                = 'k8s.gcr.io',
-  Array[String] $default_path                             = ['/usr/bin', '/usr/sbin', '/bin', '/sbin', '/usr/local/bin'],
-  String $cgroup_driver                                   = $facts['os']['family'] ? {
+  Optional[String] $docker_extra_daemon_config                   = undef,
+  Optional[String] $http_proxy                                   = $undef,
```

Comment:
@nickperry here's extra `$`, `undef` is a type, `$undef` would be a variable (but undefined :slightly_smiling_face: )

Reply:
🤦‍♂️Thank you so much! 🙂 I completely failed to see that.